sdk/servicebus/azure-servicebus/stress/scripts/dev_requirements.txt

```diff
@@ -1,4 +1,4 @@
-aiohttp>=3.0; python_version >= '3.5'
+aiohttp>=3.0; python_version >= '3.7'
```

Comment:
This python_version specifies versions for which aiohttp will install and work as expected. While technically, we no longer support Python 3.5/3.6, it's not really necessary to update this version number since we'll always be using Python 3.5+ anyway.

Though unlikely, it's also possible that we'll run these tests against a previous version of `azure-servicebus` that supported Python 3.5. In that case, we'd want to keep this as is.